src/views/Project/Details/Frameworks/FrameworkList/index.js

```diff
@@ -119,6 +137,9 @@ export default class FrameworkList extends React.PureComponent {
                             placeholder={_ts('project.framework', 'searchFrameworkInputPlaceholder')}
                             showHintAndError={false}
                             showLabel={false}
+                            // This was done so that used can continuously type
+                            // even when frameworkListPending is true
+                            disabled={false}
```

Comment:
@frozenhelium , @tnagorra should be go with this behavior?

Reply:
should remove disabled